tests/imfile-file-descriptor-exhasution.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# This is part of the rsyslog testbench, licensed under GPLv3
```

Comment:
Are you willing to change this to ASL 2.0? We are moving towards that license and I would prefer not having to remove things once (whenever this is...) we can do the switch.

Reply:
Sure, i will make the change and update. Thanks.